src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -107,6 +107,7 @@ class StableDiffusionPipeline(DiffusionPipeline, TextualInversionLoaderMixin, Lo
     _optional_components = ["safety_checker", "feature_extractor"]
     _exclude_from_cpu_offload = ["safety_checker"]
     _callback_tensor_inputs = ["latents", "prompt_embeds", "negative_prompt_embeds"]
+    _interrupt = False
```

Comment:
how about simply make it `interrupt` an pipeline property and make it available to users via callbacks? this way they can decide when to interrupt

Reply:
nice idea!